engine/src/main/java/org/terasology/logic/debug/MovementDebugCommands.java

```diff
@@ -233,22 +236,48 @@ public String stepHeight(@Sender EntityRef client, @CommandParam("height") float
         return "";
     }
 
+    private float getJumpSpeed(float amount){
+        double d = (double)amount;
+        return (float) pow(d,0.6) * 9.36f ;
+    }
+    
+    private float getInteractionRange(float amount){
+        double d = (double)amount;
+        return (float) pow(d,0.6) * 3.9f ;
+    }
+
+    private float getRunFactor(float amount){
+       return 0.15f + (0.64f * amount) - (0.006f * amount * amount);
```

Comment:
The run factor seems to be a value in the range from 0 to 10, with 1.5 being the default. 

Again, calculating the new run factor with the default height of 1.6 yields a value of 1.15864 (≠1.5).

This is how the run factor scales with the given function for `amount` between 0 and 5. The curve looks almost linear, to be hones:
![image](https://user-images.githubusercontent.com/1448874/78275222-98ff8d80-7511-11ea-9c71-f48586f3fe47.png)
see [Wolfram Alpha](https://www.wolframalpha.com/input/?i=0.15+%2B+%280.64+*+x%29+-+%280.006+*+x%5E2%29+%2C+x+in+%5B0%2C5%5D)